docs/products/kafka/kafka-connect/howto/gcp-pubsub-lite-sink.md

```diff
@@ -45,7 +45,7 @@ Apache Kafka® service page, *Overview* tab, and *Schema Registry* subtab
 
 As of version 3.0, Aiven for Apache Kafka no longer supports Confluent
```

Comment:
```suggestion
Starting with version 3.0, Aiven for Apache Kafka no longer supports Confluent
```

Reply:
This was probably added when it was only Apache Kafka version 3.0; we now support version 3.6.